cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -1,17 +1,44 @@
 #!/bin/sh
 
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ -z "${AWS_SHARED_CREDENTIALS_FILE}" ]; then echo "ENV VAR AWS_SHARED_CREDENTIALS_FILE is not set"; exit 1; fi
+
 ### expirationDate is always with timezone +00:00, eg, "2019-07-23T22:35+0000"
-MY_DATE=$(TZ=":Africa/Abidjan" date '+%Y-%m-%d')
-declare -a regions=("us-east-1" "us-east-2" "us-west-1")
+my_date=$(TZ=":Africa/Abidjan" date '+%Y-%m-%d')
+
+if [[ -n "${IPI_DEPROVISION_DATE_ARG-}" ]]; then
```

Comment:
We should validate that this is set, as without it, the script is not useful.

Reply:
nit: that's a lot of names in the env var! we could just use something like `${CLUSTER_TTL}`

Reply:
rename: Done.
validation: not sure how to validate it. Could do it as a follow-up i think.